liquibase-service/src/main/resources/db/rdb_modern/routines/017-sp_d_labtest_result_postprocessing-002.sql

```diff
@@ -0,0 +1,1450 @@
+
+        SELECT
+            lab104.lab_test_uid,
+            REPLACE(REPLACE(ovt.ovt_value_txt, CHAR(13), ' '), CHAR(10), ' ')	'LAB_RESULT_COMMENTS'  , -- asLab_Result_Comments,
+            ovt.ovt_seq	'LAB_RESULT_TXT_SEQ'  , -- AS Lab_Result_Txt_Seq,
+            lab104.record_status_cd
+        INTO #TMP_Lab_Result_Comment
+        FROM
+            #TMP_Result_And_R_Result		AS lab104
+                INNER JOIN nrt_observation_txt AS ovt ON ovt.observation_uid =  lab104.lab_test_uid
+        WHERE 	ovt.ovt_value_txt IS NOT NULL
+          AND ovt.ovt_txt_type_cd = 'N'
+          AND ovt.ovt_seq <>  0;
+
+
+        IF @pDebug = 'true' SELECT 'DEBUG: TMP_Lab_Result_Comment',* FROM #TMP_Lab_Result_Comment;
+
+
+        SELECT @ROWCOUNT_NO = @@ROWCOUNT;
+        INSERT INTO [DBO].[JOB_FLOW_LOG]
+        (BATCH_ID,[DATAFLOW_NAME],[PACKAGE_NAME] ,[STATUS_TYPE],[STEP_NUMBER],[STEP_NAME],[ROW_COUNT])
+        VALUES(@BATCH_ID,'D_LABTEST_RESULTS','D_LABTEST_RESULTS','START', @PROC_STEP_NO,@PROC_STEP_NAME,@ROWCOUNT_NO);
+        COMMIT TRANSACTION;
+
+        BEGIN TRANSACTION;
+        SET @PROC_STEP_NO =  @PROC_STEP_NO + 1;
+        SET @PROC_STEP_NAME = 'GENERATING #TMP_New_Lab_Result_Comment ';
+
+        IF OBJECT_ID('#TMP_New_Lab_Result_Comment', 'U') IS NOT NULL
+            DROP TABLE #TMP_New_Lab_Result_Comment;
+
+        SELECT *,
+               cast( NULL AS varchar(2000)) AS v_lab_result_val_comments
+        INTO #TMP_New_Lab_Result_Comment
+        FROM #TMP_Lab_Result_Comment;
+
+
+        SELECT @ROWCOUNT_NO = @@ROWCOUNT;
+
+        INSERT INTO [DBO].[JOB_FLOW_LOG]
+        (BATCH_ID,[DATAFLOW_NAME],[PACKAGE_NAME] ,[STATUS_TYPE],[STEP_NUMBER],[STEP_NAME],[ROW_COUNT])
+        VALUES(@BATCH_ID,'D_LABTEST_RESULTS','D_LABTEST_RESULTS','START', @PROC_STEP_NO,@PROC_STEP_NAME,@ROWCOUNT_NO);
+
+        COMMIT TRANSACTION;
+
+        BEGIN TRANSACTION;
+
+        SET @PROC_STEP_NO =  @PROC_STEP_NO + 1;
+        SET @PROC_STEP_NAME = 'GENERATING #TMP_New_Lab_Result_Comment_grouped ';
+
+
+        create index idx_TMP_New_Lab_Result_Comment_uid ON  #TMP_New_Lab_Result_Comment (lab_test_uid);
+
+
+        IF OBJECT_ID('#TMP_New_Lab_Result_Comment_grouped', 'U') IS NOT NULL
+            DROP TABLE  #TMP_New_Lab_Result_Comment_grouped;
+
+
+        SELECT DISTINCT LRV.lab_test_uid,
+                        SUBSTRING(
+                                (
+                                    SELECT ' '+ST1.lab_result_comments  AS [text()]
+                                    FROM #TMP_New_Lab_Result_Comment ST1
+                                    WHERE ST1.lab_test_uid = LRV.lab_test_uid
+                                    ORDER BY ST1.lab_test_uid,ST1.lab_result_txt_seq
+                                    FOR XML PATH ('')
+                                ), 2, 2000) v_lab_result_val_txt
+        INTO #TMP_New_Lab_Result_Comment_grouped
+        FROM #TMP_New_Lab_Result_Comment LRV;
+
+
+        UPDATE #TMP_New_Lab_Result_Comment
```

Comment:
@upasanapattnaik-eq Move the temp table update statement logic to the Select SQL. Across the file I see a lot of Updates that are smaller transformations